broadinstitute/orsp-pub
# BTRX- 712 - Convert role Management

## Addresses
[BTRX-712](https://broadinstitute.atlassian.net/browse/BTRX-712): ORSP - Convert Role Management

## Changes
- Remove Roles Management grails dependency.
- Implement React Router to Main Component in order to improve redirects in React.
- Use Application context path from configurations on router browser in order to dynamically change according to the environment the app is working on (dev / test / prod, etc...)
- Remove unnecessary modules export entries.

## Testing
- Navigate to Roles Management from the navigation bar. 
- Access to Projects or Sample Data Cohorts from any link, it should be correctly redirected.
- Start application from another context, e.g Test. previous redirections should still be ok.

---
